### PR TITLE
Docker/add settings

### PR DESCRIPTION
**Pull request by bvizureanu**
### What does it fix?

Closes #XXX 

the local `.env` file on dev should be appended with:

```
SWAGGER_REDIRECT_URIS=https://api.deurgenta.hostmysite.ro/swagger/oauth2-redirect.html
SWAGGER_CORS=https://api.deurgenta.hostmysite.ro
IDENTITYSERVER_URL=https://auth.deurgenta.hostmysite.ro
HANGFIRE_USERNAME=?
HANGFIRE_PASSWORD=?
```

### CODE REVIEW

File: Src/docker-compose.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@
         dockerfile: DeUrgenta.RecurringJobs.Dockerfile
       environment:
         - ASPNETCORE_URLS=http://+:80
-        - ConnectionStrings__DbConnectionString=Server=postgres;Port=5432;Database=de-urgenta-db;User Id=docker;Password=docker;
+        - ConnectionStrings__DbConnectionString=${DB_CONNECTION_STRING}
```
Comment (catileptic) [inline Src/docker-compose.yml]: I don't think this is a correct syntax. it's either 

`VAR: ${OTHER_VAR_FROM_ENV}`

or 

`- VAR=VALUE`

But not both at the same time. If you want the DB connection string to come from the `.env` file, all the vars in the `environment` settings for the API should be read from the `.env` file.

**Review by catileptic — CHANGES_REQUESTED** [review]

File: Src/docker-compose.yml (hunk 1 above)
Comment (bvizureanu) [inline Src/docker-compose.yml:39]:
```suggestion
        - ConnectionStrings__DbConnectionString: ${DB_CONNECTION_STRING}
```
so like this?

**Review by catileptic — CHANGES_REQUESTED** [review]
nope, the syntax isn't ok 

lemme help:

the `docker-compose.yml` file should become (for the `environment` part of the API service):

```
environment:
        ASPNETCORE_URLS: ${ASPNETCOTE_URLS}
        ConnectionStrings__DbConnectionString=Server: ${DB_CONNECTION_STRING}
        IdentityServerUrl: ${IDENTITY_SERVER_URL}
```

and the `.env` file would become:

```
FROM_EMAIL=noreply@deurgenta.ro
SENDGRID_API_KEY=
ASPNETCORE_URLS=http://+:80
DB_CONNECTION_STRING=Server=postgres;Port=5432;Database=de-urgenta-db;User Id=docker;Password=docker;
SWAGGER_REDIRECT_URIS=https://localhost:5040/swagger/oauth2-redirect.html
SWAGGER_CORS=https://localhost:5040
ASPNETCOTE_URLS=http://+:80
IDENTITY_SERVER_URL=https://auth.deurgenta.hostmysite.ro
```

from what i gather, we can't mix both `- var=value` and `var: ${var}`. so let's stick to one.

File: Src/docker-compose.yml (hunk 1 above)
Comment (bvizureanu) [reply Src/docker-compose.yml]: I saw is that you should not have mixed syntax for the same service
You might be able to mix them up for distinct services

Comment (tudoramariei) [reply Src/docker-compose.yml]: The syntax is valid. Both `: ${BLAH}` and `= ${BLAH}` work fine.

**Review by catileptic — APPROVED** [review]
yass! thanks, @bvizureanu !

File: Src/docker-compose.yml (hunk 1 above)
Comment (bvizureanu) [reply Src/docker-compose.yml]: we ❤️ consistency: everyone now has `key: ${env_variable}`

**Pull request merged**